data_structures/list_array.cpp

```diff
@@ -29,8 +29,9 @@ namespace list_array {
     /**
      * @brief Structure of List with supporting methods.
      */
+    template <int N>
```

Comment:
I believe we should use `uint64_t` in here like the other values.
```suggestion
    template <uint64_t N>
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable